apis/python/src/tiledbsoma/_index_util.py

```diff
@@ -3,26 +3,45 @@
 SOMATileDBContext which would otherwise ensue.
 """
 
+from typing import Optional
+
 import numpy as np
-import pandas as pd
+from somacore.query.types import IndexLike
 
 from tiledbsoma import pytiledbsoma as clib
 
 from .options import SOMATileDBContext
 
+"""
```

Comment:
docstring goes inside function, on first line, not above it.

See examples elsewhere in the code base